Frameworks/include/UIApplicationInternal.h

```diff
@@ -16,6 +16,9 @@
 #pragma once
 
 #import <UIKit/UIApplication.h>
+#import <UWP/WindowsFoundation.h>
```

Comment:
IIRC not including them in a header was intentional, something funky happened with the linker because of how our modules take ownership of headers or something to that effect.  I'd prefer sticking to forward declarations until that's resolved so someone doesn't accidentally stumble into that wasp nest